README.md

```diff
@@ -91,13 +184,108 @@ You then need to do a second call to the `signIn` mutation providing only the `o
 }
 ```
 
-## OAuth
+## OAuth / Login with Sorare
+
+If you want to make requests on behalf of other users you can rely on the [OAuth](https://oauth.net/2/) protocol.
```

Comment:
I would reword this a bit. Really this is something like:

"With our OAuth API, user's can sign-in to your service using their Sorare account, which allows you to request data on their behalf."